### PR TITLE
docs/readme: fix some grammar issues in README.md

### PR DESCRIPTION
**Pull request by abacon**
I've done a little proofreading of the README to correct some
small grammar issues and make it more readable.

### CODE REVIEW

**Comment by abacon** [comment]
Eeep! I see now that there's already some work done on this.

File: README.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ For more info refer to,
 [documentation]: http://autumnai.github.io/leaf
 
 > Disclaimer: Leaf is currently in an early stage of development.
-> If you are experiencing any bugs that are not due to not yet implemented
-> features, feel free to create a issue.
+> If you are experiencing any bugs with features that have been
+> implemented, feel free to create a issue.
```
Comment (hobofan) [inline README.md]: It was actually meant to say the opposite, but I can't really remember why, and I think your change works just as well.

**Comment by hobofan** [comment]
@abacon Thanks! :)
Yes there is some overlap with the other PRs. I am not sure how big the overlap is, but it definitely has some good changes that the other ones don't, so I'll merge it.

**Comment by hobofan** [comment]
@homu r+

**Pull request merged**